widgets/GoogleDrive-batchsource.json

```diff
@@ -488,6 +515,30 @@
         "operator": "equal to",
         "value": "oAuth2"
       },
+      "show": [
+        {
+          "name": "oauthMethod",
+          "type": "property"
+        }
+      ]
+    },
+    {
+      "name": "Authenticate with OAuth2 Access Token",
+      "condition": {
+        "expression": "oauthMethod == 'ACCESS_TOKEN' && authType == 'oAuth2'"
```

Comment:
when we upgrade older pipeline using `oAuth2`, it will not have `oAuthMethod` set, did we test this scenario?

I think the expression should be `oAuthMethod != 'REFRESH_TOKEN' && authType == 'oAuth2'`, please test it for older pipelines

Reply:
we have do a null check here in case of upgrade, added the same

Reply:
yeah I saw a null check in backend for upgrade this comment was more about UI

Reply:
on the UI only added a null check now